concepticondata/conceptlists/Huisman-2019-110.tsv-metadata.json

```diff
@@ -0,0 +1,74 @@
+{
+    "@context": [
+        "http://www.w3.org/ns/csvw",
+        {
+            "@language": "en"
+        }
+    ],
+    "dialect": {
+        "encoding": "utf-8-sig",
+        "delimiter": "\t",
+        "skipBlankRows": true
+    },
+    "tables": [
+        {
+            "tableSchema": {
+                "columns": [
+                    {
+                        "datatype": {
+                            "base": "string",
+                            "format": "[a-zA-Z]+\\-[0-9]{4}\\-[0-9]+[a-z]?\\-[0-9]+[a-z]?$"
+                        },
+                        "name": "ID"
+                    },
+                    {
+                        "datatype": {
+                            "base": "string",
+                            "format": "[0-9\\.]+([a-z\\\u2013]+)?$"
+                        },
+                        "name": "NUMBER"
+                    },
+                    {
+                        "datatype": {
+                            "base": "integer",
+                            "minimum": "1"
+                        },
+                        "name": "CONCEPTICON_ID"
+                    },
+                    {
+                        "datatype": "string",
+                        "name": "CONCEPTICON_GLOSS"
+                    },
+                    {
+                        "datatype": "string",
+                        "name": "ENGLISH"
+                    },
+                    {
+                        "datatype": "string",
+                        "name": "SWADESH"
+                    },
+                    {
+                        "datatype": "string",
+                        "name": "JAPANESE"
+                    },
+                    {
+                        "datatype": "string",
+                        "name": "ROMANISATION"
+                    },
+                    {
+                        "datatype": "string",
+                        "name": "NOTES"
+                    },
+                    {
```

Comment:
There's multiple empty columns after the `CONCEPTICON_GLOSS` row. Ideally these would be removed and the empty column definition removed here.